backend/tests/modules/auth/test_router.py

```
@@ -141,3 +204,85 @@ def test_login_invalid_email_format(client):
+    assert "not authenticated" in response.json()["detail"].lower()
+
+
+def test_login_inactive_user(client):
+    """Test login with inactive user fails."""
+    # Register a user
+    register_payload = {
+        "email": "inactive@example.com",
+        "password": "testpassword123"
+    }
+    register_response = client.post("/api/auth/register", json=register_payload)
+    user_id = register_response.json()["id"]
```

Ruff (0.14.6)

Local variable `user_id` is assigned to but never used

Remove assignment to unused variable `user_id`

(F841)